src/ArcxAnalyticsSdk.ts

```diff
@@ -253,6 +255,20 @@ export class ArcxAnalyticsSdk {
     return true
   }
 
+  /** Report error to the server in order to better understand edge cases which can appear */
+  _reportError(error: string): Promise<string> {
+    return postRequest(this.sdkConfig.url, this.apiKey, '/report-error', {
+      identityId: this.identityId,
+      error,
+    })
```

Comment:
This depends on https://github.com/arcxmoney/monorepo/pull/2057. Will merge for now but just as an FYI

Reply:
Agree, I mentioned it in pr description